src/main/java/com/ratifire/devrate/controller/InterviewController.java

```diff
@@ -39,4 +44,28 @@ public List<InterviewDto> getAll() {
   public void deleteRejected(@PathVariable long id) {
     interviewService.deleteRejected(id);
   }
+
+  /**
+   * Retrieves the details of a specific interview event by its ID.
+   *
+   * @param id the ID of the event
+   * @return an InterviewEventDto containing the event details
+   */
+  @GetMapping("/events/{id}")
+  public InterviewEventDto getInterviewEventDetails(@PathVariable long id) {
+    return interviewService.getInterviewEventDetails(id);
+  }
+
+  /**
+   * Retrieves a list of interview event short details starting from the specified date and time.
+   *
+   * @param from the starting date and time from which to retrieve interview events
+   * @return a list of InterviewEventShortDto objects containing the short details of the interview
+   *     events
+   */
+  @GetMapping("/closest")
```

Comment:
do we use 
@GetMapping()
  public List<InterviewDto> getAll() {
    return interviewService.getAll();
  }
  
  it looks like a useless endpoint.  maybe it will be better to remove it and  rename this one @GetMapping("/closest") -> @GetMapping()